postgres/fabfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+import boto3
+from invoke import Responder
+from fabric import task
+
+
+@task(name="setup")
+def setup(ctx):
+    # Setup auto-responder
+    postgres_owner_password = ctx.config.secrets.postgres_owner.password
+    postgres_passord_responder = Responder(pattern="Password", response= postgres_owner_password + "\n")
+    # Run Postgres commands with port forwarding
+    with ctx.forward_local(local_port=1111, remote_host=ctx.config.django.postgres_host, remote_port=5432):
+        # Clear all databases and application role
+        ctx.local(f'psql -h localhost -p 1111 -U postgres -W -c "DROP DATABASE edushare"',
+                  echo=True, warn=True, watchers=[postgres_passord_responder], pty=True)
+        ctx.local(f'psql -h localhost -p 1111 -U postgres -W -c "DROP DATABASE harvester"',
```

Comment:
Wat is de reden dat harvester wordt gedropt en niet weer wordt aangemaakt?

Reply:
Harvester wordt aangemaakt in de initdb.sql. De reden dat edushare daar niet wordt aangemaakt is dat Docker container altijd 1 database aanmaakt (voor ons: edushare)